nodejs18.x/full-stack/{{cookiecutter.project_name}}/frontend/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "sam-starter-fontend",
+  "version": "1.0.0",
+  "description": "Frontend app for the Innovator Island workshop.",
+  "author": "Randeep Singh <randeepx@amazon.com>, Ben Peterson <benpte@amazon.com>",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint"
```

Comment:
Can you add some unit test cases to test the frontend components, and update the unit test case defined [here](https://github.com/aws/aws-sam-cli-app-templates/blob/e5496c0dcdb17a367f3c336e9e562f6ad0735275/tests/integration/unit_test/test_unit_test_nodejs18_x.py#L47-L49) to run the forntend lint, and test commands.

Reply:
Added unit tests for frontend vue components.

Reply:
unit test cases added in the most recent commit

Reply:
updated to fix failing unit tests